packaging/docker/config.yaml

```diff
@@ -1,22 +1,4 @@
-manager:
```

Comment:
turns out all of this is now not required for building an image (ie. running with `--only-install`). Only `services_to_install` is taken into account for that.